addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/VoiceRSSTTSService.java

```diff
@@ -164,22 +164,37 @@ public AudioStream synthesize(String text, Voice voice, AudioFormat requestedFor
         return audioFormats;
     }
 
-    /**
-     * Up to now only MP3 supported.
-     */
-    private final AudioFormat getAudioFormat(String format) {
-        // MP3 format
-        if (AudioFormat.CODEC_MP3.equals(format)) {
-            // we use by default: MP3, 44khz_16bit_mono
+    private final AudioFormat getAudioFormat(String apiFormat) {
+        if (apiFormat.equals("MP3")) {
+            // we use by default: MP3, 44khz_16bit_mono with bitrate 64 kbps
             Boolean bigEndian = null; // not used here
             Integer bitDepth = 16;
-            Integer bitRate = null;
+            Integer bitRate = 64000;
             Long frequency = 44000L;
 
             return new AudioFormat(AudioFormat.CONTAINER_NONE, AudioFormat.CODEC_MP3, bigEndian, bitDepth, bitRate,
                     frequency);
+        } else if (apiFormat.equals("OGG")) {
+            // we use by default: OGG, 44khz_16bit_mono
+            Boolean bigEndian = null; // not used here
+            Integer bitDepth = 16;
+            Integer bitRate = null;
+            Long frequency = 44000L;
+
+            return new AudioFormat(AudioFormat.CONTAINER_OGG, AudioFormat.CODEC_VORBIS, bigEndian, bitDepth, bitRate,
+                    frequency);
+        } else {
+            throw new RuntimeException("Audio format " + apiFormat + "not yet supported");
```

Comment:
Better use a more specific exception, in this case `IllegalArgumentException` seems to fit

Note: that a space is missing between `apiFormat` and 'not'